frontend/src/features/projects/erp/forms/erpYupSchema.ts

```diff
@@ -1,6 +1,18 @@
 import * as Yup from 'yup';
+import _ from 'lodash';
+import { IStatus } from 'features/projects/common';
+
+/**
+ * Determine if the specified project status code requires a clearance notification sent on value.
+ * @param statusCode The project status code.
+ */
+export const clearanceNotificationSentOnRequired = (statusCode: string) => {
+  return _.includes(['ERP-ON', 'ERP-OH'], statusCode); // ERP-ON, ERP-OH
```

Comment:
/nit, try and use the es6 includes when possible. ie ['ERP-ON', 'ERP-OH'].includes(statusCode)

Reply:
also not sure if I agree with putting this check here as I think we normally just use yup schema files for yup schema. Not a big deal I suppose.

Reply:
I've moved to utils